inventory/byo/hosts.example

```diff
@@ -20,6 +20,10 @@ ansible_ssh_user=root
 # deployment type valid values are origin, online and enterprise
 deployment_type=enterprise
 
+# product type valud values are openshift and atomic-enterprise
```

Comment:
s/valud/valid/

Also, if openshift is going to be the default, this could be commented out and reference atomic-enterprise instead.